controllers/chefs/updateChefOrderStatus/index.js

```diff
@@ -0,0 +1,21 @@
+import { NotFoundError } from '../../../helpers/index.js';
+import Order from '../../../models/order/Order.model.js';
+
+export const updateChefOrderStatus = async (req, res) => {
+  const { orderId } = req.params;
+  const updateStatus = req.body;
+
+  const updatedChefOrdertatus = await Order.findByIdAndUpdate(
```

Comment:
A little misprint - updatedChefOrder tatus

Reply:
Sure, I'll fix it